python/paddle/v2/reader/creator.py

```diff
@@ -57,7 +57,7 @@ def reader():
     return reader
 
 
-def recordio_local(paths, buf_size=100):
+def recordio(paths, buf_size=100):
```

Comment:
@typhoonzero It's used as `paddle.v2.reader.creator.recordio`, probably `reader` is indicated in the import path.